src/components/PipelineRun/PipelineRunListView/PipelineRunListRow.tsx

```diff
@@ -115,6 +131,23 @@ const BasePipelineRunListRow: React.FC<React.PropsWithChildren<BasePipelineRunLi
           '-'
         )}
       </TableData>
+      <TableData className={pipelineRunTableColumnClasses.triggeredBy}>
+        {gitProvider === 'github' ? (
+          <GithubIcon className="git-icon" />
+        ) : gitProvider === 'gitlab' ? (
+          <GitlabIcon className="git-icon" />
+        ) : null}
+
+        {eventType ? (
+          <ExternalLink
+            href={`https://${gitProvider}.com/${repoOrg}/${repoURL}#${prNumber}`}
```

Comment:
This is not a valid github/gitlab url.
For Pull request pipeline runs
please create a utility to generate github/gitlab PR url, you can use label/annotation `pipelinesascode.tekton.dev/source-repo-url` to get the repo url, if that annotation is not present you can use the separate `gitprovider`, `repoOrg`, etc... to generate the repo url.

If a pipeline run is for push event, you should show the commit id that can link to commit details view.
Please get UX approval on this.